google-datacatalog-mysql-connector/src/google/datacatalog_connectors/mysql_/config/num_rows_query.sql

```diff
@@ -0,0 +1,9 @@
+SELECT  t.table_schema as database_name,
+        t.table_name, t.table_rows
+
+      FROM information_schema.tables t
+
+      WHERE t.table_schema NOT IN
+          ('mysql', 'information_schema',
```

Comment:
Do you think it makes sense to extract those internal system tables names to a constant or another config file?

the other query has the same internal system tables.

Reply:
I find it's a good idea :) but I don't know how to do it in SQL. Does it have sort of tuple/list data structure where I can store multiple strings?

Reply:
You can use the SQL file as a template SQL, and create the resolved SQL in the code.

I remember that you were working on a Query Assembler, the Query Assembler could resolve the SQL file.

if we are going to need multiple queries, and they all will have those same internal system tables, probably worth doing that.

If you are going to send other Pull Requests, I'd suggest doing this when you need another query, for the other fields you will be working on... So this Pull Request is not too big.

Reply:
for now, this is all I've got in terms of PRs :) I'll do some more work on MySQL/PostrgreSQL connectors at the end of my internship, if there is time left. Let's then postpone it until that moment in time.